### DISCUSSION TITLE
Use of identifiers for the gene symbol components of gene fusions

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#11] [Other discussion topics]
The VICC Gene Fusion guidelines include recommendations for the description of fusions by component gene symbols, subsuming the scope of the recently published HGNC guidelines on gene fusions from Bruford et al. [[1]](https://www.nature.com/articles/s41375-021-01436-6)

The VICC guidelines provide a mechanism for optionally specifying identifiers on the gene components of gene fusions:

`symbol(identifier)::symbol(identifier)` (first use of a gene symbol in a document)
`symbol::symbol` (subsequent use of gene symbol in a document)

For example: _BCR_(hgnc:1014)::_ABL1_(hgnc:76)

This pattern extends the `symbol(identifier)` syntax recommended by the HGNC for the use of gene symbols in other contexts[[2]](https://www.pnas.org/doi/full/10.1073/pnas.2025207118)[[3]](https://www.nature.com/articles/s41588-020-0669-3). However, this clashes with recent HGNC gene fusion guidelines which explicitly do not recommend identifying gene symbols when they are components of a gene fusion description:

> We do not recommend that the IDs be included in the fusion notation, but rather in the accompanying text, e.g., a gene fusion involving _BCR_(HGNC:1014) and _ABL1_(HGNC:76) is denoted as _BCR_::_ABL1_. [(source)](https://www.nature.com/articles/s41375-021-01436-6#:~:text=We%20do%20not%20recommend%20that%20the%20IDs%20be%20included%20in%20the%20fusion%20notation%2C%20but%20rather%20in%20the%20accompanying%20text%2C%20e.g.%2C%20a%20gene%20fusion%20involving%20BCR(HGNC%3A1014)%20and%20ABL1(HGNC%3A76)%20is%20denoted%20as%20BCR%3A%3AABL1.)

No justification for this is described in the HGNC gene fusion guidelines or elsewhere (to my knowledge). Given the importance of uniquely identifying gene symbols for long-term data fidelity (especially in contexts where the gene fusion is the first and/or only appearance of the constituent symbols), understanding this recommendation is paramount to aligning the VICC and HGNC recommendations[[4]](https://www.nature.com/articles/s41375-021-01458-0#:~:text=While%20there%20are,manuscript%20or%20report.).

In discussing this issue with some authors of the HGNC gene fusion guidelines, it sounded like the primary challenge may have been the use of the single colon (`:`) in the HGNC identifiers (which are [CURIEs](https://en.wikipedia.org/wiki/CURIE)) and the potential challenges that may cause alongside the use of the single colon in the ISCN nomenclature, though the specific details have not been discussed.

This discussion should include the explicit rationale behind the HGNC gene fusion recommendations against identifiers in fusion descriptions, motivated by examples, nomenclature patterns, or other technical considerations that would be confounded by the use of HGNC-style symbol identification.

**Comment by RosHastir** [comment]
I have had feedback from several different parts of the genomics community, including ISCN committee, HGNC, diagnostic laboratories, global EQA providers and specialists in haematology.

1. The proposal that the notation for a fusion gene should include the identifier i.e. BCR(hgnc:1014)::ABL1(hgnc:76) is not consistent with ISCN, HGVS or the paper published by Bruford et al.. 2020.
2. The  addition of identifier e.g. hgnc:1014 make the notation/nomenclature cumbersome and feedback from the ISCN committee and diagnostic laboratories is that this would not be universally adopted. 
3. The genetics community is aware of the genes involved in the fusions and therefore there is no need for a duplication of information and/or the inclusion of a HGNC identifier in the notation as it doesn't add any additional information nor does it increase clarity.  
4. This change would require a major upheaval of internal reporting systems and databases. 
5. The inclusion of the identifier is not an essential requirement of the notation. This identifier information can be given in the text of the publication or diagnostic report if needed.  This is consistent with other identifiers used in genomic testing.  The current ISCN & HGVS nomenclature includes the genome build so the gene names are correct within this context.
6. Finally, the use of a colon in the identifier infers a break in ISCN and therefore the ISCN committee strongly advised against the inclusion of a colon in the identifier.


Ros Hastings
Chair of the ISCN Committee

**Reply by microkim1** [reply]
The fusions are very important in hematologic malignancies. They are used in WHO classification Tumours of Haematopietic and Lymphoid Tissues. The recent publication does not include the identifier. And we should be aware that the gene fusions are used for pathologists and hematologists as well as  genetic specialists. Therefore, it is better to be simple. 

![image](https://user-images.githubusercontent.com/123788125/215366574-afcc739a-8f9c-4c44-b722-c759bf0724d2.png)

Myungshin
ISCN Committee member

**Reply by ahwagner** [reply]
Thanks @RosHastir and @microkim1 for this detailed feedback, we will be considering these arguments for revising our gene symbol identifier recommendations.

**Comment by elspethb** [comment]
As per the statement in the HGNC recommendations for the designation of gene fusions, and from further subsequent discussions with co-authors of this article, we would not endorse the use of HGNC IDs within the gene fusion notation in publications. 

HGNC do indeed advise quoting stable gene identifiers, ideally HGNC IDs which are arguably the most stable human gene identifiers, at least once within an article that references any specific human gene, to enable data retrieval and archiving. This guidance also applies when describing gene fusions - that the gene IDs be quoted within the article, but _not_ within the notation. We feel it would make the notation significantly harder to read and interpret, and far easier to make mistakes, if the gene fusion included any form of gene IDs. We are certain that part of the reason the proposed standard has gratifyingly already been widely adopted in publications is its simplicity, which would not be the case were it to include brackets, gene IDs (which users would have to check) and additional colons.  HGNC IDs were primarily created for data exchange, and are not designed to be user friendly. 

Please note the publication from Bruford et al was simply proposing a new user-friendly standard predominantly for use in publications - we were not suggesting a reporting standard, and specifically stated "Full ISCN [[14](https://www.nature.com/articles/s41375-021-01436-6#ref-CR14)] (https://iscn.karger.com/) or HGVS (https://varnomen.hgvs.org/recommendations/DNA/variant/complex/) nomenclature should be used for formal reporting."

**Reply by ahwagner** [reply]
@elspethb, thank you for this detailed feedback–it is very informative and helps me understand the HGNC recommendations much more clearly.
